Library/Homebrew/cmd/audit.rb

```diff
@@ -203,6 +203,7 @@ def audit_file
     return unless @strict
 
     component_list = [
+      [/^  include Language::/,            "include method"],
```

Comment:
I'm afraid `include method` is going to be confusing, as `foo method` tends to refer to `def foo` (see below). How do you feel about referring to it as simply `include` or maybe `include directive`?